### DISCUSSION TITLE
Pull head/tail image files from https://media.battlesnake.com

### DISCUSSION THREAD

**Discussion by chris-bsnake** [BattlesnakeOfficial/feedback#43] [Feature Requests]
Currently, all the image files representing the heads and tail customizations for Battlesnakes are baked directly into the Exporter build. These files exist on the Battlesnake CDN (https://media.battlesnake.com) and shouldn't have to be copy-managed or require Exporter deploys every time a new customization is add into the platform.  Ideally, the exporter could pull these files from CDN and cache them locally.

**Comment by bvanvugt** [comment]
Created an issue (#72) to track this.
